src/app/legacy/components/Footer/Link/index.jsx

```diff
@@ -17,22 +18,48 @@ const StyledLink = styled.a`
   }
 `;
 
-const Link = ({ service, text, href, inline, lang }) => (
-  <StyledLink
-    service={service}
-    lang={lang}
-    inline={inline}
-    href={href}
-    className="focusIndicatorInvert"
-  >
-    {text}
-  </StyledLink>
-);
+const Link = ({
+  service,
+  text,
+  href,
+  inline,
+  lang,
+  onClick,
+  onlyShowIfJSenabled,
+}) => {
+  const [isVisible, setVisible] = useState(onlyShowIfJSenabled !== true);
```

Comment:
Can this just be `useState(onlyShowIfJSenabled)`. Should always be a boolean value so I don't think we need to compare it here.

Reply:
It can be `! onlyShowIfJSenabled` if you prefer?

Reply:
Ah I see, I think thats ok as is 👍